specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/stable/2021-11-12/hanaonazure.json

```diff
@@ -0,0 +1,707 @@
+        },
+        "description": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The localized friendly description for the operation as shown to the user. This description should be thorough, yet concise. It will be used in tool-tips and detailed views."
+        },
+        "origin": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The intended executor of the operation; governs the display of the operation in the RBAC UX and the audit logs UX. Default value is 'user,system'"
+        }
+      },
+      "description": "Detailed HANA operation information"
+    },
+    "ErrorResponse": {
+      "type": "object",
+      "description": "Describes the format of Error response.",
+      "properties": {
+        "error": {
+          "type": "object",
+          "description": "Describes the error object.",
+          "properties": {
+            "code": {
+              "readOnly": true,
+              "description": "Error code",
+              "type": "string"
+            },
+            "message": {
+              "readOnly": true,
+              "description": "Error message indicating why the operation failed.",
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "Tags": {
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Tags field of the HANA instance."
+        }
+      },
+      "description": "Tags field of the HANA instance."
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Subscription ID which uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call."
+    },
+    "ResourceGroupNameParameter": {
+      "name": "resourceGroupName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Name of the resource group.",
+      "x-ms-parameter-location": "method"
+    },
+    "HanaInstanceNameParameter": {
+      "name": "hanaInstanceName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Name of the SAP HANA on Azure instance.",
+      "x-ms-parameter-location": "method"
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Client API version."
+    },
+    "TagsParameter": {
+      "name": "tagsParameter",
+      "in": "body",
+      "required": true,
+      "schema": {
+        "$ref": "#/definitions/Tags"
+      },
+      "description": "Request body that only contains the new Tags field",
```

Comment:
The diff b/w this file and json file for preview version shows some unexpected deletions from this line forward related to monitoring details, why is that so?

Reply:
fixed